backend/routes/institution/institution_router.py

```
@@ -0,0 +1,421 @@
+        return ResponseModel(status="success", message=msg)
+    except Exception as e:
+        logger.error(f"Error deleting team: {e}")
+        return ErrorResponseModel(
+            status="error", message=f"Failed to delete team: {str(e)}"
+        )
+
+
+@institution_router.get("/get-all-teams", response_model=ResponseModel)
+@verify_admin_or_institution
+async def get_teams_endpoint(
+    session: Session = Depends(get_db),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)